### ISSUE TITLE
Failing on valid 'extern crate mylib;'

### ISSUE THREAD

**Issue by welbornprod** [oschwald/SublimeLinter-contrib-rustc#12]
It complains about this line at the top of my file:

```rust
extern crate spattools;
```

It says "Error: can't find external crate \`spattools\`"
It's a valid library and cargo/rustc compiles my project just fine. Is there some setting that I am missing?

**Comment by oschwald** [comment]
Do you have `use-cargo` enabled?

**Comment by welbornprod** [comment]
That's what it was. I missed the ["linters"](http://sublimelinter.readthedocs.org/en/latest/settings.html#linter) section of SublimeLinter's config help, and I missed it in the actual config file too. Not enough sleep I guess. 

Sorry for the bogus report.

**Comment by oschwald** [comment]
No problem!
